.gitignore

```diff
@@ -6,4 +6,5 @@ dist/*
 public/build/*
 public/encrypt/*
 public/index.css
+public/locales.json
```

Comment:
This is a built file that contains all the locales. We don't need it in the repo.